### PR TITLE
remove old junk, replace index.html with a troika-styled version

### PR DESCRIPTION
**Pull request by haakoneriksen**
screenshot from 2017-12-20 12-25-06

### CODE REVIEW

**Review by mortenlj — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,4 +1,2 @@
-finn.github.com
```
Comment (theneva) [inline README.md:1]: lol

**Review by theneva — APPROVED** [review]

**Comment by ivarconr** [comment]
Niceness.  Would it be possible to have logos as well with the featured projects? E.g. Unleash logo is here: https://github.com/Unleash/unleash-frontend/blob/master/public/logo.png

Btw we don't do projects in FINN, call them products/solutions/whatever instead?

**Comment by haakoneriksen** [comment]
Sure, I just didn't know we had logos for stuff. Will fix!
We discussed the term "project" in the policy meetings, and I think we decided that "an open-source project" is a very established term which doesn't necessarily denote a body of work with a beginning and an end, but rather "a piece of software which is opensource"

**Comment by gregersrygg** [comment]
Sorry for sticking my nose into this (got a notification for some reason), but you might want to minify and concat those stylesheets 😉  38 stylesheets for 2 small webpages seems a bit crazy. Not that speed really matters on these pages, but this page is to show off, right?

**Comment by haakoneriksen** [comment]
@gregersrygg  How DARE you comment on this code that we made publicly available on the internet?! 😃 
There are a lot of the CSS stuff that's not even used, so I'll clean it up and at least minify them.

**Comment by SimenB** [comment]
the css files all have a `.min` version

**Pull request merged**